platform/podjs/src/browserPod.ts

```diff
@@ -29,55 +29,10 @@ const NAV_LIGHT_FOREGROUND_COLOR = "#ffffff";
 
 const MANIFEST_DATA = window.manifestData;
 
-// Previously, data were stored in localStorage. IndexedDB is more flexible and
-// efficient, and most importantly in our case can store larger amount of data,
-// while the 2MB limit of localStorage is no longer sufficient. This function
-// migrates any data in localStorage to not disrupt the developer expirience.
-// TODO: Please remove the migration code after Oct 2022
-async function migrateData(db: IDBDatabase): Promise<IDBDatabase> {
-    const polyInData = localStorage.getItem("polyInStore");
-    const polyOutData = localStorage.getItem("files");
-
-    if (polyInData || polyOutData) {
-        const files: FileInfo[] = [];
-        const quads = polyInData
-            ? JSON.parse(polyInData).map(RDFString.quadToStringQuad)
-            : [];
-
-        if (polyOutData) {
-            for (const [, file] of JSON.parse(polyOutData)) {
-                const dataUrl = localStorage.getItem(file.id);
-                if (dataUrl) {
-                    files.push({
-                        id: file.id,
-                        name: file.name,
-                        time: new Date(file.time),
-                        blob: await (await fetch(dataUrl)).blob(),
-                    });
-                }
-            }
-        }
-
-        await new Promise((resolve, reject) => {
-            const storeNames = [OBJECT_STORE_POLY_IN, OBJECT_STORE_POLY_OUT];
-            const tx = db.transaction(storeNames, "readwrite");
-            const polyInStore = tx.objectStore(OBJECT_STORE_POLY_IN);
-            for (const quad of quads) polyInStore.add(quad);
-            const polyOutStore = tx.objectStore(OBJECT_STORE_POLY_OUT);
-            for (const file of files) polyOutStore.add(file);
-            tx.oncomplete = () => resolve(null);
-            tx.onerror = tx.onabort = () => reject(tx.error);
-        });
-
-        localStorage.clear();
-    }
-
-    return db;
-}
-
-function openDatabase(): Promise<IDBDatabase> {
+async function openDatabase(): Promise<IDBDatabase> {
     return new Promise((resolve, reject) => {
-        const request = indexedDB.open(DB_NAME, DB_VERSION);
+        const name = `${DB_PREFIX}:${document.location.pathname}`;
```

Comment:
I think this is a good approach. I mean, we can go down two routes here:

1. We focus on providing a workaround for Chrome not assigning different origins to different `file:` URLs - this logic would produce unique strings in this scenario, you cut of the protocol, but that part is identical anyway. But logically, we might as well use the full URL here.
2. We basically _abuse_ the behaviour of Chrome, giving the same feature the same storage no matter where its HTML file is located. For this, we could cut use `manifest.name` - which we can determine from the URL as a fallback.

But I'm for (1), honestly.